### ISSUE TITLE
Description of Dataset

### ISSUE THREAD

**Issue by diggerdu** [diggerdu/icassp2018-cranberry#2]

**Comment by Alexuan** [comment]
Randomly chosen 2142 utterances from the VCTK training set are used for training (5 speakers × 2 genders ×
231 utterances). Short-time Fourier transform with an 1024-point frame size and a 50% overlap is used for the
time-frequency conversion. Resilient backpropagation (Rprop) [25] technique is employed, and their parameters are
found through a validation with additional four speakers: 0.5, 1.5, 10−7
and 10−1
for backtracking, acceleration, and
minimum and maximum step sizes, respectively. We choose a modified Rectified Linear Unit (ReLU) as proposed
in [14] for the activation function. Dropout parameters p
(l)
are all set to be 0.8. The sum of the sq
